test/kustomize/kustomize.go

```diff
@@ -15,10 +20,14 @@ func Build(dir string) (*bytes.Buffer, error) {
 	cmd.Stdout = &stdout
 	cmd.Stderr = &stderr
 
+	fmt.Println(cmd.String())
+
 	err := cmd.Run()
 
-	fmt.Println(stdout.String())
-	fmt.Println(stderr.String())
+	if logOutput {
+		fmt.Println(stdout.String())
```

Comment:
These two prints can be replaced with:

``
o, _ := cmd.CombinedOutput()
fmt.Printf("%s", string(o))
``